CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## Unreleased
+## 0.2.0.2 (Mar 2023)
 
 * `toChunks` and `toBytes` now make the stdin available to the process being
```

Comment:
Isn't this a breaking change?
Can you diff the previous release using packdiff and see what other changes are there in this release?